Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2370,6 +2370,8 @@ extension BrowserViewController: SessionRestoreHelperDelegate {
         if let tab = tabManager.selectedTab, tab.webView === tab.webView {
             updateUIForReaderHomeStateForTab(tab)
         }
+
+        clipboardBarDisplayHandler?.didRestoreSession()
```

Comment:
Previously this hooked into a notification; when I updated session restore code I saw this notification wasn't needed, we have a direct way to call `clipboardBarDisplayHandler?.didRestoreSession()`